website/src/data/gallery.json

```diff
@@ -0,0 +1,155 @@
+[
+  {
+    "title": "AutoGen Playground",
+    "link": "https://huggingface.co/spaces/thinkall/AutoGen_Playground",
+    "description": "A space to explore the capabilities of AutoGen.",
```

Comment:
be more precise. @thinkall please double check.

Reply:
resolved